src/dl_patch_ptr.cpp

```diff
@@ -1,6 +1,30 @@
 #include "dl_patch_ptr.h"
 #include "dl_types.h"
 
+struct dl_patched_ptrs
```

Comment:
This is a plain "undo" of the things mentioned here: https://github.com/wc-duck/datalibrary/pull/158/files#r1000718654